javascript/test/ParserTest.ts

```diff
@@ -308,5 +309,128 @@ description
 
       assert.strictEqual(pickle.steps[0].argument.docString.content, '```what')
     })
+
+    it("parses Markdown data tables with headers", () => {
```

Comment:
This is a re-implementation of the `datatables.md` acceptance test - once issues have been resolved then this test can probably be removed.